dynamic-layers/virtualization/recipes-azure-iot/iotedge/aziot-edged/aziot-edged/fix_compiler_toolchain_1.8x.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/edgelet/docker-rs/Cargo.toml b/edgelet/docker-rs/Cargo.toml
```

Comment:
the name of the patch is confusing ... you don't patch a toolchain.
the fix is solving lint issues (my guess: regardless of the rust version >= 1.8x)

Reply:
file name changed. 
Idea to generate only one patch doesn't make sense; location of patch content is different.